cmd/admin.go

```diff
@@ -146,6 +146,37 @@ var (
 		Name:   "list",
 		Usage:  "List auth sources",
 		Action: runListAuth,
+		Flags: []cli.Flag{
+			cli.IntFlag{
+				Name:  "min-width",
+				Usage: "Minimal cell width including any padding for the formatted table",
+				Value: 0,
+			},
+			cli.IntFlag{
+				Name:  "tab-width",
+				Usage: "width of tab characters in formatted table (equivalent number of spaces)",
+				Value: 8,
+			},
+			cli.IntFlag{
+				Name:  "padding",
+				Usage: "padding added to a cell before computing its width",
+				Value: 1,
+			},
+			cli.StringFlag{
+				Name: "pad-char",
+				Usage: `ASCII char used for padding
+				if padchar == '\\t', the Writer will assume that the
```

Comment:
Won't this text be unintentionally indented? AFAICT the tabs will be part of the string.

Reply:
Probably best to just let you have a play with some code to see what the variables do:

https://play.golang.org/p/zD6eXOjd8U8

Reply:
oh I see what you're talking about!